### DISCUSSION TITLE
What are domain alias?

### DISCUSSION THREAD

**Discussion by codebude** [the-djmaze/snappymail#1122] [Q&A] [answered]
In the Admin panel in the `Domains` section there is a button `Add Alias`. What does it do? In the snappy wiki, the paragraph for this option is not maintained: https://github.com/the-djmaze/snappymail/wiki/Admin-Manual#add-alias

Can anybody enlighten me?

**Comment by cm-schl** [comment, marked as answer]
Hi,
thanks for pointing that out!
I've added this to the documentation: https://github.com/the-djmaze/snappymail/wiki/Admin-Manual#add-alias

**Reply by codebude** [reply]
Thanks for the clarification!
